inductiva/fluids/scenarios/wind_tunnel/README.md

```diff
@@ -0,0 +1,151 @@
+- `get_output_mesh`: Returns the mesh of the airflow over the entire domain and data on the object.
+- `get_object_pressure_field`: Returns the pressure field of the airflow over the object.
+- `get_streamlines`: Input parameters - `max_time`, `n_points`, `initial_step_length`, `source_radius`, `source_center`. Returns a mesh of the streamlines with pressure and velocity components of airflow over the domain.
+- `get_flow_slice`: Input parameters - `plane`, `origin`. Returns a mesh of the flow slice with pressure and velocity components.
+- `get_force_coefficients`: Returns the force coefficients that represent the forces acting on the object. These are the drag and lift coefficients.
+
+For the pressure field, streamlines and flow slices there are easy-to-use visualizations, which have some configuration parameters. See the example below for the general overview of these parameters.
+
+The next example fetches the default post-processed files and renders the respective visualizations. The last one, fetches the full simulation output, and with the same interface the user can configure the post-processing tools to extract the metrics he desires and render the visualizations.
+
+### Example for default files:
+
+```python
+# Get the default files from WindTunnel simulation
+output = task.get_output()
+
+# Get a pressure_field mesh
+pressure_field = output.get_object_pressure_field()
+
+# Render
+pressure_field.render_frame()
+```
+
+<img src="../../../../resources/media/openfoam/default_pressure_field.png" width="400" height="300" />
+
+```python
+# Get a mesh of the streamlines 
+streamlines = output.get_streamlines()
+
+# Render the streamlines
+streamlines.render_frame(physical_field="velocity",
+                         flow_cmap="viridis",
+                         view="isometric",
+                         streamline_radius=0.1
+                         save_path="default_streamlines.png")
+```
+
+<img src="../../../../resources/media/openfoam/default_streamlines.png" width="400" height="300" />
+
+
+```python
+# Get a mesh of the flow slice
+flow_slice = output.get_flow_slice(plane="xz")
+
+# Render the pressure field over the slice
+flow_slice.render_frame(physical_field="pressure",
+                        flow_cmap="viridis",
+                        save_path="default_flow_slice.png")
+```
+
+<img src="../../../../resources/media/openfoam/default_flow_slice.png" width="400" height="300" />
+
+### Example for general Post-processing
+
+```python
+# Get all the WindTunnel simulation files
+output = task.get_output(full=True)
+
+# Get a pressure field mesh
+pressure_field = output.get_object_pressure_field()
+
+# Render
+pressure_field.render_frame(save_path="pressure_field.png")
+```
+
+<img src="../../../../resources/media/openfoam/pressure_field.png" width="400" height="300" />
+
+```python
+# Get the streamlines mesh
+streamlines = output.get_streamlines(max_time=200,
```

Comment:
Here is a general comment, for future, and I think you need to address this with @hpenedones 
I think if you hang everything in the output object this sort of objects becomes overly complex, and they also will become difficult to document. And, worse, their logic will vary imensly from simulator to simulator.

For now, let's not change anything BUT I think we should consider moving to a world where the output objects are basically interfaces between the result data and the users, and they basically know about where data lives. They don't have to do any specialized processing: they are just interfaces to data, they know where data is located, and they are able to move data around.

Once you have one of those output object, then you can basically pass it to a PostProcessing object (which can transform, visualize, combine, etc the data). For example, here I would prefer to have something like>

```
output = task.get_output()

streams = StreamLinesProcessor(output)
# At this point StreamLinesProcessor doesn't care about where data lives.
# but the output object knows where data lives in the server. So, even visualization
# methods can be run on the server.
streams.plot("name.png", other_params=...)
```
This pattern can be repeated for every scenario.....

@hpenedones @fabiocruz @cristianacarpinteiro @david451lima : wdyt? Do you see where I am trying to go?

Reply:
This is a good idea @sarmento. However, the way streamlines (or other data) is defined in data files varies from simulator to simulator (e.g. they may be saved in different file formats, or defined differently within a common file format), so we'd still have to code a streamlines processor for each scenario.